test/access/AccessControl.test.js

```diff
@@ -17,6 +17,15 @@ describe('AccessControl', function () {
     this.accessControl = await AccessControlMock.new({ from: admin });
   });
 
+  describe('_setupRole', function () {
+    it('cannot be called outside the constructor', async function () {
```

Comment:
Is there a test that it _can_ be used in the constructor?

Reply:
Yes, the mock uses it on construction.